<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileChoose</class>
 <widget class="QWidget" name="FileChoose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>195</width>
     <height>100</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="twinkleButton">
      <property name="text">
       <string>小星星</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="fireworksButton">
      <property name="text">
       <string>打上花火</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QPushButton" name="oceanButton">
      <property name="text">
       <string>我们都拥有海洋</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="fightButton">
      <property name="text">
       <string>Fight Song</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="queren">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>90</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>90</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>95</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="create">
      <property name="text">
       <string>开始创作</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="chooseButton">
      <property name="text">
       <string>选择模板</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="open">
      <property name="text">
       <string>打开文件</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
